bindgen/codegen/mod.rs

```diff
@@ -2595,8 +2595,16 @@ impl MethodCodegen for Method {
             })
         }
 
-        let block = quote! {
-            #( #stmts );*
+        let block = if ctx.options().wrap_unsafe_ops {
```

Comment:
Looks good but maybe we can add a `ctx.wrap_unsafe(TokenStream) -> TokenStream` that does this to avoid the duplication?